packages/flutter_tools/templates/create/.travis.yml.tmpl

```diff
@@ -0,0 +1,11 @@
+os:
+  - osx
+osx_image: xcode7.2
+sudo: false
+before_script:
+  - git clone https://github.com/flutter/flutter.git /tmp/flutter
```

Comment:
I edited it back. Though it is a bit disconcerting that anyone can make edits to comments and have those be attributed to someone else.